docker-compose.yml

```diff
@@ -1,9 +1,28 @@
-version: '3.7'
+version: '2.2'
 services:
 
+  setup:
```

Comment:
this is interesting.....      I wonder though, the pattern in chorus has been that allof this kind of setup/validation logic, plus the "timing" logic, has all been in the `quickstart.sh`...   is it simpler this way, or more confusing to an end user?

Or is it that this logic here doesn't matter, so shouldn't be in quickstart...   that it's just plubming...   humm...

Reply:
I've taken that directly from Elastic's documentation and adapted it to our use case. The only purpose of this is to set a password for a user (`kibana_system`) which was not necessary prior to Elasticsearch 8.x.
I guess we could add it to the `quickstart.sh` script but it really is just setting up a password for a system user that is only used within Kibana. As a user I don't really want to be bothered with these kinds of things and having them set automatically (without any kind of scripting) would be personal preference. So my argument is that it is easier if we have it in the compose file as it decreasess the steps that need to be taken to set everything up.

Reply:
Makes sense...     Plus if that is how they recommend it.....